src/farcasterd/trade_state_machine.rs

```diff
@@ -1137,6 +1137,7 @@ fn attempt_transition_to_end(
                                         break;
                                     } else {
                                         warn!("{} | Auto-funding Monero transaction failed with {}, retrying, {} retries left", &swap_id.bright_blue_italic(), err, retries);
+                                        tokio::time::sleep(std::time::Duration::from_secs(retries)).await;
```

Comment:
I fear this introduces too much of a delay. In the worst case, it blocks the entire daemon for 55 seconds. If you really need this, I would at the least lower this to either 10's or 100's of ms.

Reply:
> In the worst case, it blocks the entire daemon for 55 seconds.

Don't exaggerate: it's only 45! :P

Reduced to 10ms increments in https://github.com/farcaster-project/farcaster-node/pull/733/commits/82764f45e7216701c8548f569f573a55851064c0, so upper bound 450 ms. If this catches most failures within the first two retries, let's cut by another order of magnitude.